### ISSUE TITLE
Some way to filter your library

### ISSUE THREAD

**Issue by Malikil** [Reyder95/sdvx-tracker-frontend#133]
As your library gets bigger, it would be nice to have way of filtering the songs in it by things like last played, clear type, number of scores, or other song things like level or game.

Even if those things aren't actually displayed, like I do think the current layout does a good enough job for what it feels like it's showing. Though splitting into pages or doing an infinite scroll system like the users page does might be nice.

**Comment by Reyder95** [comment]
Yeah these are planned. I just kinda threw the library together. I want to do things like things as I slowly redesign all the tables and things
